### ISSUE TITLE
트레이닝 데이터 정제

### ISSUE THREAD

**Issue by shin285** [shineware/KOMORAN#125]
## 기능 설명
admin 툴을 이용하여 어휘 관련 트레이닝 데이터 정제

**Comment by shin285** [comment]
@songyonge 사전에 잘못된 정보가 굉장히 많이 포함되어 있어서 정제를 하려고합니다. 혹시 가능하실까요?

**Comment by songyonge** [comment]
어드민 툴이라 함은 Komoran Admin 말씀하시는 거죠?
잘못된 정보가 "굉장히" 많다고 하시니, 우선 순위를 정해야 할 것 같군요.

**Comment by shin285** [comment]
@songyonge 네 맞습니다. Komoran Admin입니다만, 현재 Komoran Admin이 KOMORAN에 통합되었기 때문에 master branch를 clone 받으시면 Admin을 실행시키실 수 있습니다.

**Comment by songyonge** [comment]
@shin285 grammar.in에서 불필요해보이는 것들을 삭제하였습니다.
[komoran_grammar.in_20200610.txt](https://github.com/shineware/KOMORAN/files/4755742/komoran_grammar.in_20200610.txt)

**Comment by songyonge** [comment]
@shin285 역시 트레이닝 코퍼스나 참고 데이터 없이 사람의 직관에 따라서 하는 작업은 문제가 있네요.
말씀하신 예시 중에 "약해져"의 경우에는 "약하/VA+어/EC+지/VX+어/EC"로 분석되어야 할 것 같은데요.
누락되었던 문법 규칙과는 무관해보입니다.
